### DISCUSSION TITLE
Any legal issues for me if I do not transfer client's site ownership?

### DISCUSSION THREAD

**Discussion by thedamruta** [plausible/analytics#2360] [Anything Else]
I have created an account at plausible, booked the subscription plan and not only I want to track my own website data, but as well as the client's.
So when I set up everything for the client, I should transfer ownership of the site and then he becomes responsible for the payment for his site.
My question is - could I keep as well the client's site under my account and do not transfer the ownership, so only that I would be paying for the plan? Does it have any legal implications for me in the future?
Just wondering, since I can write the plan price to my work expenses and have 50 sites under my plausible account, is it crucial that every client does buy it's own analytics plan? Or I will get into huge legal trouble if I do not encourage my clients to do so?

Kind regards,
Ruta

**Comment by metmarkosaric** [comment]
hi @thedamruta! you can do either way (keep the ownership or transfer the ownership) so feel free to choose whatever you and your clients prefer. for us it's all good

**Reply by thedamruta** [reply]
thank you, Marco! :)
